core/types/block.libevm.go

```diff
@@ -29,8 +29,8 @@ import (
 // HeaderHooks are required for all types registered with [RegisterExtras] for
 // [Header] payloads.
 type HeaderHooks interface {
-	MarshalJSON(*Header) ([]byte, error) //nolint:govet // Type-specific override hook
-	UnmarshalJSON(*Header, []byte) error //nolint:govet
+	EncodeJSON(*Header) ([]byte, error) // Type-specific override hook
```

Comment:
```suggestion
	EncodeJSON(*Header) ([]byte, error)
```

That was a comment explaining the `nolint`.